app/src/main/java/com/example/movopfy/features/movies/presentation/viewmodel/MoviesViewModel.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.movopfy.features.movies.presentation.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.movopfy.features.movies.domain.repository.KinopoiskRepository
+import com.example.movopfy.network.kinopoisk.models.KinopoiskDocs
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+const val HORRORS = "Ужасы"
+const val COMEDY = "Комедия"
+const val DRAMA = "Драма"
+const val MELODRAMA = "Мелодрама"
+const val HORRORS_CATEGORY = "+ужасы"
+const val COMEDY_CATEGORY = "комедия"
+const val DRAMA_CATEGORY = "драма"
+const val MELODRAMA_CATEGORY = "!мелодрама"
+
+class MoviesViewModel(private val kinopoiskRepository: KinopoiskRepository) : ViewModel() {
+
+    private val _uiState =
+        MutableStateFlow<MoviesUiState>(MoviesUiState.Loading)
+    val uiState: StateFlow<MoviesUiState> = _uiState.asStateFlow()
+
+    private val list = arrayListOf<KinopoiskDocs>()
+    private var page = 1
+
+    private fun categoryList(category: String) = when (category) {
```

Comment:
that method is doesn`t belong to ViewModel. It should be like util, extension or mapper

Reply:
I did